lib/mount.js

```diff
@@ -22,11 +22,21 @@ const mount = (app, { errLogger, logger }={}) =>
       .then(log(logger, { req, res }))
 
 const write = (req, res) => ({ body, headers={}, statusCode=200 }) => {
-  res.statusCode = statusCode
-
   for (var name in headers)
     res.setHeader(name, headers[name])
 
+  const requestEtag = req.headers['if-none-match']
+  const bodyEtag = !(body instanceof Stream) ? etag(body || '') : undefined
```

Comment:
It's not possible to return an `etag` for a streamed response, since the headers are sent before the body begins streaming.  So this is just fine.  My only comment is that the `undefined` is unnecessary.  You get the same behavior with this:
```js
const bodyEtag = !(body instanceof Stream) && etag(body || '')
```
That way `bodyEtag` is either `false` or a string.